website/pages/docs/plugins/destinations/bigquery/types.mdx

```diff
@@ -0,0 +1,51 @@
+| Int8                   | ✅ Yes        | `INT64`                                                  |
+| Int16                  | ✅ Yes        | `INT64`                                                  |
+| Int32                  | ✅ Yes        | `INT64`                                                  |
+| Int64                  | ✅ Yes        | `INT64`                                                  |
+| Interval[DayTime]      | ✅ Yes        | `RECORD<days: INT64, milliseconds: INT64>`               |
+| Interval[MonthDayNano] | ✅ Yes        | `RECORD<months: INT64, days: int64, nanoseconds: int64>` |
+| Interval[Month]        | ✅ Yes        | `RECORD<months: INT64>`                                  |
+| JSON                   | ✅ Yes        | `JSON`                                                   |
+| Large Binary           | ✅ Yes        | `BYTES`                                                  |
+| Large List             | ❌ No         |                                                          |
+| Large String           | ✅ Yes        | `STRING`                                                 |
+| List                   | ✅ Yes        | (Repeated column) †                                      |
+| MAC                    | ✅ Yes        | `STRING`                                                 |
+| Map                    | ❌ No         |                                                          |
+| String                 | ✅ Yes        | `STRING`                                                 |
+| Struct                 | ✅ Yes        | `RECORD`                                                 |
+| Timestamp[ms]          | ✅ Yes        | `TIMESTAMP`                                              |
+| Timestamp[ns]          | ✅ Yes        | `RECORD<timestamp: TIMESTAMP, nanoseconds: INT64>`       |
+| Timestamp[s]           | ✅ Yes        | `TIMESTAMP`                                              |
+| Timestamp[us]          | ✅ Yes        | `TIMESTAMP`                                              |
+| UUID                   | ✅ Yes        | `STRING`                                                 |
+| Uint8                  | ✅ Yes        | `INT64`                                                  |
+| Uint16                 | ✅ Yes        | `INT64`                                                  |
+| Uint32                 | ✅ Yes        | `INT64`                                                  |
+| Uint64                 | ✅ Yes        | `NUMERIC`                                                |
+| Union                  | ❌ No         |                                                          |
+
+## Notes
+
+† Repeated columns in BigQuery do not support null values. Right now, if an array contains null values, these null values will be dropped when writing to BigQuery. Also, because we use REPEATED columns to represent lists, lists of lists are not supported right now.
```

Comment:
```suggestion
† Repeated columns in BigQuery do not support null values. Right now, if an array contains null values, these null values will be dropped when writing to BigQuery. Also, because we use `REPEATED` columns to represent lists, lists of lists are not supported right now.
```